### PR TITLE
Add CloudRaft as ADOPTERS

### PR DESCRIPTION
**Pull request by anjuls**
### What this PR does
Before this PR:

After this PR:

Adds [CloudRaft](https://cloudraft.io/) to adopters.

We are using KubeVirt in a GPU cloud platform that allows end users to run VMs built for AI workload. This allows cost-effective experimentation and provides access to powerful GPUs. 

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
add cloudraft to adopters.
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 | End-User | arm | 2021 | [link](https://www.arm.com) | KubeVirt enables seamless transition from legacy Virtual Machine based workloads to cloud-native container platforms. Arm believes KubeVirt addresses this challenge, allowing Virtual Machines workloads to easily deploy and scale in the cloud and at the edge. <br><br>Arm is an active contributor to the project, focused on enabling and optimizing Kubevirt performance on aarch64 and working with the ecosystem to facilitate users in deploying their workloads on cloud-native platforms. |
 | End-user| Bytedance | 2023 | [link](https://www.bytedance.com/en/) | We use KubeVirt as part of our innovation Trusted Container Stack for TEE cluster provisioning. |
 | End-user| Civo | 2020 | [link](https://www.civo.com) | We are using KubeVirt as part of our stack to enable tenant cluster provisioning within Civo cloud. |
+| End-User | CloudRaft| 2024 | [link](https://cloudraft.io/) | CloudRaft is using KubeVirt to build a GPU Cloud platform for AI workload. KubeVirt allows end users such as Data scientists to experiment on powerful remote VMs equipped with GPUs and AI/ML packages.  |
```
Comment (fabiand) [inline ADOPTERS.md:12]: Are end-users ordering VMs? Then I believe that you'd qualify as a vendor, not end-user.

Comment (anjuls) [reply ADOPTERS.md:12]: @fabiand actually we use in the product and solutions. Do you suggest I should change to vendor? Happy to do it.

Comment (fabiand) [reply ADOPTERS.md:12]: If you are using the product to run your own workloads, then end-user is fine.

**Comment by aburdenthehand** [comment]
/lgtm
Welcome @anjuls and CloudRaft!

**Comment by vladikr** [comment]
> /lgtm Welcome @anjuls and CloudRaft!

+1 very happy to see this.

**Comment by dhiller** [comment]
/approve

/hold for other maintainers to chime in

@aburdenthehand @fabiand I haven't found anything wrt adding adopters formally, but I think maintainer ack is sufficient, right?

**Comment by dhiller** [comment]
/override pull-kubevirt-goveralls

Service failure

**Comment by dhiller** [comment]
/unhold

Welcome @anjuls and CloudRaft!

**Review by dhiller — APPROVED** [review]

**Pull request merged**